src/context/cart-context.js

```diff
@@ -0,0 +1,25 @@
+import { createContext, useContext, useEffect, useReducer } from 'react'
+
+const CartContext = createContext();
+
+const CartContextProvider = ({children}) => {
+    const cartReducer = (state, action) => {
+        switch (action.type) {
+            case "ADD_TO_CART":
+                return {...state, cart: [...state.cart, {...action.payload, qty:1}]};
+            case "REMOVE_FROM_CART":
```

Comment:
you can make constant file with diff exports e.g.  ``` export const ADD_TO_CART = "ADD_TO_CART " ``` for reusability and easy imports

Reply:
Will create a new PR for it, as i have to change the naming from all the files.